vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html

```diff
@@ -64,6 +65,19 @@
 }
 
 $(document).ready(function($) {
+    
+    var dateFormat = 'YYYY-DD-MM HH:mm:ss';
+    var data = [[${module.creationDate}]];
+    var month = data.monthValue<10 ? "0"+data.monthValue : data.monthValue;
+    var day = data.dayOfMonth<10 ? "0"+data.dayOfMonth : data.dayOfMonth;
+    var hour = data.hour<10 ? "0"+data.hour : data.hour;
+    var minute = data.minute<10 ? "0"+data.minute : data.minute;
+    var second = data.second<10 ? "0"+data.second : data.second;
+    var date = data.year+"-"+month+"-"+day+" "+hour+":"+minute+":"+second;
+    var testDateUtc = moment.utc(date);
```

Comment:
what does the creation date look like that you need to do all of this? moments should do the formatting for you.

Reply:
The creationDate is read as date object in the javascript.
{
  "offset": {
    "totalSeconds": -25200,
    "id": "-07:00",
    "rules": {
      "fixedOffset": true,
      "transitions": [],
      "transitionRules": []
    }
  },
  "monthValue": 6,
  "year": 2020,
  "month": "JUNE",
  "dayOfMonth": 30,
  "dayOfWeek": "TUESDAY",
  "dayOfYear": 182,
  "nano": 504000000,
  "hour": 11,
  "minute": 25,
  "second": 0
}
I could not find the function in the moment library to format the date object.
Any leads, please?

Reply:
Have you tried reading it as a string in js? I believe it already comes in a format that moments can read.